classes/class-wc-connect-payment-methods-store.php

```diff
@@ -104,11 +104,11 @@ protected function get_payment_methods_from_response_body( $response_body ) {
 				return new WP_Error( 'payment_methods_type', 'Expected but did not receive array for payment_methods.' );
 			}
 
-			foreach ( (array) $payment_methods as $payment_method ) {
+			foreach ( $payment_methods as $payment_method ) {
```

Comment:
I agree to remove this `(array)` cast since we already have a `is_array` check at line 103. Seems redundant. I wonder why we did that.

Reply:
Whoever wrote this code first seemed to make it very verbose style and add lots of extra checks.